hadrons/functions.py

```diff
@@ -224,12 +226,12 @@ def ks_initial_IonTracks(
         "IC_angle_rad": theta_rad,
     }
 
-    extra_params_dic = {
-        "unit_length_cm": grid_size_um * 1e-4,
-        "track_radius_cm": track_radius_cm,
-        "SHOW_PLOT": SHOW_PLOT,
-        # "PRINT_parameters": PRINT_parameters,
-    }
+    # extra_params_dic = {
```

Comment:
why commented out ?

Reply:
Removed since it was unused, the values are inlined when we call single_track_PDESolver.